tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java

```diff
@@ -45,398 +36,508 @@
-        URL lra = lraClient.startLRA(null, "SpecTest#getAllLRAs", LRA_TIMEOUT_MILLIS, ChronoUnit.MILLIS);
-        List<LRAInfo> lras = lraSPI.getLRAs(null);
+    public void getAllLRAs() throws WebApplicationException {
+        URL lra = lraClient.startLRA(null, lraClientId(), lraTimeout(), ChronoUnit.MILLIS);
 
-        assertNotNull(getLra(lras, lra.toExternalForm()), "getAllLRAs: getLra returned null", null);
+        List<LRAInfo> allClientLRAs = lraManagement.getLRAs(null);
+        boolean isLraIdInList = containsLraId(allClientLRAs, lra);
 
-        lraClient.closeLRA(lra);
+        assertTrue("LRA '" + lra + "' should be listed in the list of the all LRAs, as it was not closed yet, but it isn't",
+                isLraIdInList);
 
-        return PASSED_TEXT;
+        lraClient.closeLRA(lra);
     }
 
-    //    @Test
-    private void getRecoveringLRAs() throws WebApplicationException {
+    @Test
+    @Ignore
+    public void getRecoveringLRAs() throws WebApplicationException {
         // TODO
     }
 
+    /**
+     * Started LRA should in state 'active'.
+     */
     @Test
-    private String isActiveLRA() throws WebApplicationException {
-        URL lra = lraClient.startLRA(null, "SpecTest#isActiveLRA", LRA_TIMEOUT_MILLIS, ChronoUnit.MILLIS);
+    public void isActiveLRA() throws WebApplicationException {
+        URL lra = lraClient.startLRA(null, lraClientId(), lraTimeout(), ChronoUnit.MILLIS);
 
-        assertTrue(lraSPI.getStatus(lra).isActive(), null, null, lra);
+        assertTrue("LRA '" + lra + "' is not denoted as active even it was started", lraManagement.getStatus(lra).isActive());
 
         lraClient.closeLRA(lra);
-
-        return lra.toExternalForm();
     }
 
-    // the coordinator cleans up when canceled
+    /**
+     * Canceled LRA should in state 'compensated'.
+     * NOTE: the coordinator cleans up when canceled
+     */
     @Test
-    private String isCompensatedLRA() throws WebApplicationException {
-        URL lra = lraClient.startLRA(null, "SpecTest#isCompensatedLRA", LRA_TIMEOUT_MILLIS, ChronoUnit.MILLIS);
+    @Ignore
+    public void isCompensatedLRA() throws WebApplicationException {
+        URL lra = lraClient.startLRA(null, lraClientId(), lraTimeout(), ChronoUnit.MILLIS);
 
         lraClient.cancelLRA(lra);
 
-        assertTrue(lraSPI.getStatus(lra).isCompensated(), null, null, lra);
-
-        return lra.toExternalForm();
+        assertTrue("LRA '" + lra + "' is not denoted as compensated even it was canceled", lraManagement.getStatus(lra).isCompensated());
     }
 
-    // the coordinator cleans up when completed
+    /**
+     * Closed LRA should in state 'completed'.
+     * NOTE: the coordinator cleans up when completed
+     */
     @Test
-    private String isCompletedLRA() throws WebApplicationException {
-        URL lra = lraClient.startLRA(null, "SpecTest#isCompletedLRA", LRA_TIMEOUT_MILLIS, ChronoUnit.MILLIS);
+    @Ignore
```

Comment:
Ditto (I forget why I disabled some tests but the ones you have marked @Ignore will need adding to issue #92 )

Reply:
+1